docs/src/pages/getting-started/support/support.md

```diff
@@ -0,0 +1,144 @@
+
+Receive the latest news on Material-UI.
+
+[Follow us](https://twitter.com/MaterialUI)
+
+### Supported versions
+
+The Material-UI CE (Community Edition) has been 100% open-source (MIT) since the very beginning, and always will be. Developers can ensure Material-UI is the right choice for their React applications through Material-UI’s community maintenance strategy. The Material-UI team regularly ships new releases, bug fixes, and is very welcoming to community pull requests.
+
+Given the reality of time and resource restraints as well as the desire to keep innovating in the frontend development space, over time it becomes necessary for the Material-UI team to shift focus to newer versions of the framework. However, Material-UI will do everything it can to make the transition to newer versions as smooth as possible, including but not limited to publishing migration guides such as [this one for Material-UI v4](/guides/migration-v3/). The open-source community is always welcome to submit new features and bug fixes as well.
+
+The current status of each Material-UI framework version is:
+
+- Material-UI v4 (hooks) **[Active]**: Latest version in active development.
+- Material-UI v3 (browser version support): Only handle security fixes.
+- Material-UI v2 (never existed).
+- Material-UI v1 (rewrite): Inactive.
+- Material-UI v0.x: Inactive.
+
+For teams and organizations that require additional support for any framework version, Material-UI has [options available](#enterprise).
+
+## Professional help (Premium)
+
+### Enterprise
+
+Accelerate your company’s application development with a secure, long-term solution that’s optimized for enterprise and backed by Material-UI experts.
+
+- **Long Term Support.** ([LTS](https://en.wikipedia.org/wiki/Long-term_support)) provides developers and organizations with additional stability and reliability in the applications they’re developing, including guaranteed support level agreements (SLAs) and advisory services. Take the preparation time you need before upgrading to the latest active version (v4) of Material-UI or remain confident in your application’s long-term success by staying on whichever version you’re most comfortable with. LTS is available as a paid service, it extends to Material-UI v1 implementations and higher.
+- **Version Migration Assistance**
+- **Priority Fixes**
+
+For more information, [get a quote](mailto:enterprise@material-ui.com) from our team.
+
+### Advisory
+
+Supercharge your team’s development with Material-UI Advisory:
+
+#### Developer Advisory
+
+Everything your team needs to make sure you’re using React the best way as you evaluate and build on Material-UI.
+
+- Unlimited technical support covering Material-UI usage of the API, component design, performance, and architectural best practices.
```

Comment:
I have found this value proposing in Apollo offering, Ionic seems to offer the same, but use different wording. I would imagine that it can be sustainable at a high enough pricing point. Anyway, changing the chaning the wording:

> A private, dedicated Slack channel with access to our consultants.
  Get real-time advice, guidance, and feedback when you need it, with live Slack access and direct private chat with Material-UI’s expert technical team.